src/base/net/smtp.cpp

```diff
@@ -163,16 +163,26 @@ void Smtp::sendMail(const QString &from, const QString &to, const QString &subje
     }
 
     // Connect to SMTP server
+    QStringList smtpServer;
```

Comment:
`smtpServer` should have `QString` type like before.

Reply:
I used QStringList type as I wanted store the result of QString::split(). I intended `smtpServer` list to hold server hostname and port as two separate items. Saved the result of split so that I didn't have to repeat the `split` part for host and port. Would you prefer that I repeat it twice? or maybe change the name to something else (say `smtpServerParts`)?

Reply:
>or maybe change the name to something else

Yes, I would suggest rename it to `QStringList serverEndpoint`.
Also I would like to see a variable for server address: `const QString serverAddress = serverEndpoint[0];`

Reply:
Done.